resources/assets/js/components/Fields/JsTree.vue

```diff
@@ -91,7 +93,10 @@
                 tree.jstree('open_all');
 
                 let modified_tree = tree.jstree(true).get_json('#', {flat: true});
-                serverBus.$emit('formatted', modified_tree);
+                console.log(field.attr.length, field.attr.key);
+                if(field.attr.key !== undefined && field.attr.key === 'admin_menu') {
```

Comment:
I don't know if this is still relevant, but a few years ago, checking for undefined this way, caused some issues.
That's why, I always stick to this - `typeof field.attr.key !== 'undefined'`

Probably can ignore, added this comment, just so you know